### PR TITLE
[FRAME-138]: Allow Multiple Stellar Slugs

### PR DESCRIPTION
**Pull request by tarecord**
This is a major upgrade that lets plugins define multiple `stellar_slugs`. The use case came from TEC where they load the library from a shared library for several of their plugins. Registering multiple stellar_slugs will allow TEC to manually add opt-in modals for each add-on using the shared library.

Here's a walkthrough screencast of the changes: http://p.tri.be/v/oQXjX4

## Progress Tracking
- [x] Update Modal rendering to handle multiple stellar_slugs
- [x] Update Modal arguments to handle passed stellar_slug
- [x] Update `Status::set_status()` ~~and `Status::get_status()`~~ to handle multiple stellar_slugs (add slug parameter to define which slug it's updating/~~getting~~)
- [x] Update `Exit_Interview` arguments to handle passing stellar_slug
- [x] Update `Exit_Interview::plugin_action_links()` to output deactivate link for each plugin utilizing a specific stellar_slug
- [x] Update `Telemetry::get_user_details()` to handle multiple stellar_slugs
- [x] Update Telemetry Starter settings page to show/control each slug's modal
- [x] Update plugin uninstall to handle multiple stellar_slugs
- [x] Add Documentation for all new/changed features

### CODE REVIEW

**Review by ChrisMKindred — APPROVED** [review]
This looks good, I have a few quick comments. I'd like to have TEC do a quick review/integration from this branch before this gets merged just to make sure we covered everything they needed.

File: README.md [hunk 1]
```diff
@@ -105,14 +122,14 @@ When a user deletes the plugin, WordPress runs the method from `Uninstall` and c
 ### Prompting Users on a Settings Page
 On each settings page you'd like to prompt the user to opt-in, add a `do_action()`. _Be sure to include your defined stellar\_slug if you are using one_.
 ```php
-do_action( 'stellarwp/telemetry/{stellar_slug}/optin' );
+do_action( 'stellarwp/telemetry/optin', '{stellar_slug}' );
```
[the remark this answers is not in the document]
Comment (tarecord) [reply README.md:125]: No, I've left the original filter for now. Where do you think the deprecation note should be placed? In the documentation or with the `add_action` linked below?
https://github.com/stellarwp/telemetry/blob/697a5dab1a3b4f79a6469c3626a6082230cb4e49/src/Telemetry/Opt_In/Opt_In_Subscriber.php#L31-L32

[the remark this answers is not in the document]
Comment (ChrisMKindred) [reply README.md:125]: Both places would be great.

**Review by Camwyn — APPROVED** [review]
This is working for me in my local TEC branch so 👍🏼 from that front 🎉

**Review by ChrisMKindred — APPROVED** [review]
👍

**Pull request merged**